applications/FluidDynamicsApplication/python_scripts/fluid_solver.py

```diff
@@ -223,6 +212,27 @@ def _ExecuteCheckAndPrepare(self):
 
         check_and_prepare_model_process_fluid.CheckAndPrepareModelProcess(self.main_model_part, prepare_model_part_settings).Execute()
 
+    def _SetAndFillBuffer(self):
```

Comment:
@rubenzorrilla if you want to make it consistent with the structural app then I suggest to refactor this fct from there to the [auxiliary_solver_utilities](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/auxiliary_solver_utilities.py) and then use it from both apps

Reply:
Great.

Reply:
Makes much sense. I'll proceed as suggested.

Reply:
Done. See #9594.

Reply:
This can be further extended for adjoint buffer filling as well. I'll take care of it. :)